run0/src/cli/mod.rs

```diff
@@ -0,0 +1,52 @@
+mod run;
+
+use crate::cli::run::RunCommand;
+use clap::{Parser, Subcommand};
+
+/// `Handler` is a trait that should be implemented for each of our commands.
+///
+/// It defines the contract & the input / output of a command execution.
+pub trait Handler {
+    /// Executes the command handler.
+    ///
+    /// Every command should take no argument, has it is built at runtime with these arguments.
+    /// Also, a command must always return a `Result<()>`.
+    fn handler(&self) -> crate::Result<()>;
+}
+
+#[derive(Parser, Debug)]
+#[clap(version, author)]
+pub struct Cli {
+    /// Container bundle
+    #[clap(subcommand)]
+    pub(crate) command: Commands,
+}
+
+impl Cli {
+    /// Get the command used by the user.
+    ///
+    /// For example, if the user executes the command `run`,
+    /// we dynamically return the command so the `main` can
+    /// execute it.
+    pub fn get_command(self) -> Box<dyn Handler> {
```

Comment:
Please rename this getter to `command`. See https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#gettersetter-apis